internal/runtime/core.go

```diff
@@ -0,0 +1,31 @@
+package runtime
+
+import (
+	"awans.org/aft/internal/datatypes"
+	"awans.org/aft/internal/db"
+	"awans.org/aft/internal/starlark"
+	"fmt"
+)
+
+type FunctionHandle interface {
+	Invoke(arg interface{}) (interface{}, error)
+}
+
+func CodeToFunctionHandle(c db.Code) FunctionHandle {
+	var fh FunctionHandle
+	switch c.Runtime {
+	case db.Golang:
+		fh = &datatypes.GoFunctionHandle{Function: c.Function}
```

Comment:
this is still binding together a bunch of stuff -- I'm inclined to say keep this as it is for now, and maybe I can take a crack and factoring the runtimes out a bit more down the line.

Reply:
Sounds good. It's likely I don't have what you're thinking in mind just yet